chapters/meta-information.adoc

```diff
@@ -252,3 +252,112 @@ servers:
         default: .dev
 <...>
 ----
+
+[#253]
```

Comment:
Nice, I'm pretty happy with this.

Just one note, maybe in your example, mention that an integer **would** be good in this circumstance with another example.

Reply:
yeah I thought of that too but then I thought I have mentioned in the description above that it is all about strings
Plus after we merge the other PR, I plan to add location of all the valid and invalid example specification files
And integer being valid is part of one of the unit test scenario..